packages/config/src/layer2s/optimism.ts

```diff
@@ -84,6 +84,11 @@ export const optimism: Layer2 = {
         sinceTimestamp: new UnixTime(1636654763),
       },
     ],
+    transactionApi: {
+      type: 'rpc',
+      provider: 'alchemy',
+      networkName: 'optimism',
+    },
```

Comment:
I would prefer the name of this PR be after this change. The logging is just a small improvement. Or this could be moved to another PR.

Reply:
Moving this to a separate PR